test/tests/auxkernels/higher_order/high_order_over_sample.i

```diff
@@ -0,0 +1,80 @@
+# MOOSE Test Suite
+#-----------------
+#
+# This test demonstrates the usage of higher order auxilary variables. A standard Gaussian function is is calulated on
+# an eighth order monomail basis on a coarse mesh (4x4). Constant or linear elements would be insufficient to capture
+# the curvature of the solution given the number of elements. Oversampling with four levels of refinemint is used to visualize
```

Comment:
How about:
```
@requirement Higher order auxiliary variables
```
Similar to what we do in the code already... Just a syntactic sugar.